app/src/components/ArtworkItem.tsx

```diff
@@ -1,67 +1,64 @@
 import * as React from 'react';
-import ArtworkInfo from './ArtworkInfo';
+import ArtworkInfo, { Artwork } from './ArtworkInfo';
 import TransferArtifact from './TransferArtifact';
 import ConsignArtifact from './ConsignArtifact';
 import { ContractProps } from '../helper/eth';
 import ButtonGroup from 'react-bootstrap/ButtonGroup';
+import ArtworkCard from './ArtworkCard';
 
 interface ArtworkItemProps extends ContractProps {
   tokenId: number;
-  isOwnedArtifact: boolean;
+  ownedArtifact?: true;
+  fullscreen?: true;
 }
 
-type ArtworkItemState = {
-  artwork: any;
-}
+const ArtworkItem: React.FC<ArtworkItemProps> = ({contracts, accounts, tokenId, ownedArtifact, fullscreen}) => {
+  const [artwork, setArtwork] = React.useState<Artwork>();
+  const registry = contracts.ArtifactRegistry;
 
-class ArtworkItem extends React.Component<ArtworkItemProps, ArtworkItemState> {
-  componentDidMount (): void {
-    const registry = this.props.contracts.ArtifactRegistry;
-    registry.getArtifactForToken(this.props.tokenId)
+  React.useEffect(() => {
+    registry.getArtifactForToken(tokenId)
       .then((artworkData: any) => {
         console.log(artworkData);
         const artwork = {
           metaUri: artworkData[1],
         };
-        this.setState({ artwork: artwork });
+        setArtwork(artwork);
       })
       .catch((err: any) => { console.log(err); });
-  }
+  });
 
-  render (): React.ReactNode {
-    if (!this.state) {
-      return 'Loading...';
-    }
+  if (!artwork) {
+    return <ArtworkCard img='https://file.globalupload.io/HO8sN3I2nJ.png'/>;
+  }
 
-    console.log('Artwork ' + JSON.stringify(this.state.artwork));
+  console.log('Artwork', artwork);
```

Comment:
Was therebefore but should probably remove logging statement

Reply:
I'm actually pro this logging statement as it really highlights when you mucked up the reloading